### ISSUE TITLE
MNIST data in torchvision is corrupt!

### ISSUE THREAD

**Issue by Student2Pro** [pytorch/pytorch#73480]
### 🐛 Describe the bug

I am training a RNN and find that there are huge elements in the training data Tensor of MNIST dataset. However, they cannot be printed. I test it with these codes using VS Code. The output just don't make sense. These huge numbers will be 'printable' after some matmul, and cause errors in my program.

```python
cx = data.abs() > 1000.0
if cx.any:
    mcx = data.abs().max()
    print('max: {}'.format(mcx))
    print('The huge number in data is: {}'.format(data[cx]))
    print('found error!\n')
```
The output:
```python
max: 1.0
The huge number in data is: tensor([])
found error!
```

I'm certain that I have normalized those data. Full version of the test codes are as below.
```python
import torch
from torchvision import datasets, transforms


batch_size = 64
test_batch_size = 128
shuffle_train = True
shuffle_test = True

use_cuda = torch.cuda.is_available()

device = torch.device("cuda:0" if use_cuda else "cpu")
#os.environ["CUDA_VISIBLE_DEVICES"] = "1"
kwargs = {'num_workers': 1, 'pin_memory': True} if use_cuda else {}

train_loader = torch.utils.data.DataLoader(
    datasets.MNIST('datasets/mnist', train=True, download=False,
                transform=transforms.Compose([
                    transforms.ToTensor(),
                    transforms.Normalize((0.,), (1.,))
                ])),
    batch_size=batch_size, shuffle=shuffle_train, **kwargs)
test_loader = torch.utils.data.DataLoader(
    datasets.MNIST('datasets/mnist', train=False, transform=transforms.Compose([
                    transforms.ToTensor(),
                    transforms.Normalize((0.,), (1.,))
                ])),
    batch_size=test_batch_size, shuffle=shuffle_test, **kwargs)


for batch_idx, (data, target) in enumerate(train_loader):
    print('batch {}'.format(batch_idx))

    cx = data.abs() > 1000.0
    if cx.any:
        mcx = data.abs().max()
        print('max: {}'.format(mcx))
        print('The huge number in data is: {}'.format(data[cx]))
        print('found error!\n')
```

### Versions

PyTorch version: 1.10.0
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.3 LTS (x86_64)
GCC version: (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.31

Python version: 3.8.12 (default, Oct 12 2021, 13:49:34) [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.11.0-27-generic-x86_64-with-glibc2.17
Is CUDA available: True
CUDA runtime version: 11.3.109
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 2080 Ti
Nvidia driver version: 465.19.01
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.21.4
[pip3] numpydoc==1.1.0
[pip3] torch==1.10.0
[pip3] torchattacks==3.2.2
[pip3] torchaudio==0.10.0
[pip3] torchtext==0.11.0
[pip3] torchvision==0.11.1
[conda] blas 1.0 mkl
[conda] cudatoolkit 11.3.1 h2bc3f7f_2 https://mirrors.bfsu.edu.cn/anaconda/pkgs/main
[conda] mkl 2021.4.0 h06a4308_640
[conda] mkl-service 2.4.0 py38h7f8727e_0
[conda] mkl_fft 1.3.1 py38hd3c417c_0
[conda] mkl_random 1.2.2 py38h51133e4_0
[conda] mypy_extensions 0.4.3 py38_0
[conda] numpy 1.21.4 pypi_0 pypi
[conda] numpy-base 1.21.2 py38h79a1101_0
[conda] numpydoc 1.1.0 pyhd3eb1b0_1
[conda] pytorch 1.10.0 py3.8_cuda11.3_cudnn8.2.0_0 https://mirrors.bfsu.edu.cn/anaconda/cloud/pytorch
[conda] pytorch-mutex 1.0 cuda https://mirrors.bfsu.edu.cn/anaconda/cloud/pytorch
[conda] torchattacks 3.2.2 pypi_0 pypi
[conda] torchaudio 0.10.0 py38_cu113 https://mirrors.bfsu.edu.cn/anaconda/cloud/pytorch
[conda] torchtext 0.11.0 pypi_0 pypi
[conda] torchvision 0.11.1 py38_cu113 https://mirrors.bfsu.edu.cn/anaconda/cloud/pytorch

**Comment by kshitij12345** [comment]
I think `cx.any` always evaluates to True as you are checking truthiness of the bound method. It should be `cx.any()`.

**Comment by Student2Pro** [comment]
> I think `cx.any` always evaluates to True as you are checking truthiness of the bound method. It should be `cx.any()`.

Ops, u r right.
